overrides/default.nix

```diff
@@ -2097,6 +2097,29 @@ lib.composeManyExtensions [
         }
       );
 
+      pendulum = prev.pendulum.overridePythonAttrs (
+        old: (
+          # NOTE: Versions <3.0.0 is pure Python and is not PEP-517 compliant,
+          #       which means they can not be built using recent Poetry versions.
+          if lib.versionOlder old.version "3"
+          then { }
+          else
+            lib.optionalAttrs (!old.src.isWheel or false) {
```

Comment:
You don't need both the `if then else` **and** `lib.optionalAttrs`, just roll the `versionOlder` condition in to the first argument of `lib.optionalAttrs`. Feel free to parenthesize if that makes rolling it in easier.

Reply:
Fixed it up.

Reply:
Hey @cpcloud ~ thanks for the input (this was my first time going deeper into Nix). Is there anything that I should do from my end now, seeing that this change is already in the branch?